std-bits/table/src/main/java/org/enso/table/parsing/DateParser.java

```diff
@@ -0,0 +1,46 @@
+package org.enso.table.parsing;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Locale;
+import org.enso.table.data.column.builder.object.Builder;
+import org.enso.table.data.column.builder.object.ObjectBuilder;
+import org.enso.table.parsing.problems.InvalidFormatProblemAggregator;
+
+public class DateParser extends TypeParser<InvalidFormatProblemAggregator> {
```

Comment:
that's a lot of DRY?

Reply:
Is it though? It's mostly boilerplate. Any suggestions how to reduce it? 

I guess I can reduce `makeBuilderWithCapacity` and `makeProblemAggregator` by adding a common superclass implementing these.

But for `parseSingleValue`, I don't see a decent way that is not an overkill, especially as the logic is rather simple.

Reply:
Ok I actually figured out a way.